presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java

```diff
@@ -246,7 +246,7 @@ private Environment getEnvironment()
                     exposePort(container, 5007); // debug port
                 }
 
-                if (Boolean.getBoolean("CONTINUOUS_INTEGRATION")) {
+                if (System.getenv("CONTINUOUS_INTEGRATION") != null) {
```

Comment:
`== "true"`?

Reply:
See https://github.com/prestosql/presto/blob/e88c8e7d8437eddcb805fe434a0170a43e441b51/presto-testng-services/src/main/java/io/prestosql/testng/services/FlakyTestRetryAnalyzer.java#L106 for the intent.

Reply:
and also https://github.com/prestosql/presto/pull/5936#discussion_r522031023